testdata/p4_16_samples_outputs/stack2.p4

```diff
@@ -0,0 +1,16 @@
+#include <core.p4>
```

Comment:
This should be either a positive or a negative test. If it is negative it should have a good error message.